tools/wptrunner/wptrunner/executors/executorcontentshell.py

```diff
@@ -95,6 +94,10 @@ def _read_text_block(self, deadline=None):
             if line.endswith(self.eof_marker):
                 result += line[:-len(self.eof_marker)]
                 break
+            elif line.endswith('#EOF\r\n'):
+                result += line
+                result += 'Got a CRLF-terminated #EOF - this is a driver bug.'
```

Comment:
This should be:

result += line[:-len('#EOF\r\n')]
logger.warning('Got a CRLF-terminated #EOF - this is a driver bug.')

Reply:
I was thinking here to have the result log the full line so a developer can see what is being sent, or do you think it's fine to leave this out as it could be implied what is being sent?

Reply:
Personally, I'm of the opinion that it's not worth having this special case for `#EOF\r\n`. AFAICT, `content_shell` [does not return `\r`](https://source.chromium.org/chromium/chromium/src/+/main:content/web_test/browser/test_info_extractor.h;l=44;drc=4e1b7bc33d42b401d7d9ad1dcba72883add3e2af;bpv=0;bpt=0). If wptrunner is inserting `\r` before it goes into `stdout_queue`, then we need to fix that bug instead of dealing with the alternative marker downstream here.

Reply:
https://source.chromium.org/chromium/chromium/src/+/main:third_party/blink/tools/blinkpy/web_tests/port/driver.py;drc=4d3216caa7ad9b2dbf00316c6cfa6d76fd012edb;l=693

Content shell has the code to handle '\r\n'. Doing this the same way as rwt could save us some headache.